pandas/tests/window/conftest.py

```diff
@@ -47,3 +55,35 @@ def center(request):
 @pytest.fixture(params=[None, 1])
 def min_periods(request):
     return request.param
+
+
+@pytest.fixture(params=[True, False])
+def parallel(request):
+    """parallel keyword argument for numba.jit"""
+    return request.param
+
+
+@pytest.fixture(params=[True, False])
+def nogil(request):
+    """nogil keyword argument for numba.jit"""
+    return request.param
+
+
+@pytest.fixture(params=[True, False])
+def nopython(request):
+    """nopython keyword argument for numba.jit"""
+    return request.param
+
+
+@pytest.fixture(
+    params=[
+        pytest.param(
+            "numba",
+            marks=pytest.mark.skipif(not _HAVE_NUMBA, reason="numba is not installed"),
```

Comment:
```suggestion
            marks=pytest.mark.skip_if_no("numba"))
```

Should be equivalent and do away with need for import at top of module. We already do this in other places:

https://github.com/pandas-dev/pandas/blob/6c910333f6e766c2390e515b3214fbd879c87c41/pandas/tests/io/test_html.py#L91